app/models/pacbio/library.rb

```diff
@@ -43,6 +43,12 @@ class Library < ApplicationRecord
     accepts_nested_attributes_for :primary_aliquot
 
     after_create :create_used_aliquot
+
+    # Before updating the record, the `check_volume` method is called.
+    # This is a callback that checks if the volume has been changed and
+    # if the new volume is greater than the used volume.
+    before_update :check_volume
```

Comment:
Doing a before_update here is a good idea. We can check with long read to see if this is acceptable. It would also be worth checking whether we can add this to aliquotable. The more stuff we can put in the concern the better.